public/app/core/logs_model.ts

```diff
@@ -356,7 +356,8 @@ export function logSeriesToLogsModel(logSeries: DataFrame[]): LogsModel | undefi
       const tsNs = timeNanosecondField ? timeNanosecondField.values.get(j) : undefined;
       const timeEpochNs = tsNs ? tsNs : time.valueOf() + '000000';
 
-      const messageValue: unknown = stringField.values.get(j);
+      // In edge cases, this can be undefined. If undefined, we want to replace it with empty string.
+      const messageValue: unknown = stringField.values.get(j) ?? '';
```

Comment:
Out of curiosity - this will also cast `null` to an empty string. Is it expected behaviour? Previously `null` was converted to `"null"` by `JSON.stringify()` below.

Reply:
Yeah, I was actually thinking about it and I thought it it might be better if the value is null -> for it to be empty string rather than 'null' string. But no strong opinions here. What do you think?